src/js/helpers/address-validation.js

```diff
@@ -0,0 +1,15 @@
+import WAValidator from 'wallet-address-validator'
+
+export function validateAddress (address) {
```

Comment:
make it accept asset code as parameter also

for now, it will invalidate any non-btc/eth address

Reply:
ok